### PR TITLE
Add tests for losetup package

### PR DESCRIPTION
**Pull request by mxpv**
Signed-off-by: Maksym Pavlenko <makpav@amazon.com>

*Description of changes:*
Adds some tests

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: snapshotter/pkg/losetup/losetup_test.go [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/docker/go-units"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestLosetup(t *testing.T) {
```
Comment (IRCody) [inline snapshotter/pkg/losetup/losetup_test.go]: These are good happy path test cases but I think we could add some non-happy path case like:

* Removing/detaching a device that doesn't exist
* Attaching a device from a non-existent (or malformed?) image path
* Finding devices that don't exit

Comment (mxpv) [reply snapshotter/pkg/losetup/losetup_test.go]: Done

File: snapshotter/pkg/losetup/losetup_test.go [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+	t.Run("AttachLoopDevice", func(t *testing.T) {
+		dev1, err := AttachLoopDevice(imagePath)
+		require.NoError(t, err)
+		require.NotEmpty(t, dev1)
+
+		dev2, err := AttachLoopDevice(imagePath)
+		assert.NoError(t, err)
+		assert.NotEqualf(t, dev2, dev1, "should attach different loop device")
+
+		_, err = AttachLoopDevice("")
```
Comment (samuelkarp) [inline snapshotter/pkg/losetup/losetup_test.go]: This should be a completely separate test.

File: snapshotter/pkg/losetup/losetup_test.go [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+		loopDevice2 = dev2
+	})
+
+	t.Run("FindAssociatedLoopDevices", func(t *testing.T) {
+		devices, err := FindAssociatedLoopDevices(imagePath)
+		assert.NoError(t, err)
+		assert.Lenf(t, devices, 2, "unexpected number of attached devices")
+		assert.ElementsMatch(t, devices, []string{loopDevice1, loopDevice2})
+
+		devices, err = FindAssociatedLoopDevices("")
```
Comment (samuelkarp) [inline snapshotter/pkg/losetup/losetup_test.go]: This should be a completely separate test.

File: snapshotter/pkg/losetup/losetup_test.go [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+		assert.NoError(t, err)
+		assert.Empty(t, devices)
+	})
+
+	t.Run("DetachLoopDevice", func(t *testing.T) {
+		err := DetachLoopDevice(loopDevice2)
+		require.NoErrorf(t, err, "failed to detach %q", loopDevice2)
+
+		err = DetachLoopDevice("")
```
Comment (samuelkarp) [inline snapshotter/pkg/losetup/losetup_test.go]: This should be a completely separate test.

File: snapshotter/pkg/losetup/losetup_test.go [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+
+	t.Run("RemoveLoopDevicesAssociatedWithImage", func(t *testing.T) {
+		err := RemoveLoopDevicesAssociatedWithImage(imagePath)
+		assert.NoError(t, err)
+
+		devices, err := FindAssociatedLoopDevices(imagePath)
+		assert.NoError(t, err)
+		assert.Empty(t, devices)
+
+		err = RemoveLoopDevicesAssociatedWithImage("")
```
Comment (samuelkarp) [inline snapshotter/pkg/losetup/losetup_test.go]: This should be a completely separate test.

**Pull request merged**